app/(dashboard)/mailboxes/[mailbox_slug]/[category]/conversation/messageActionButtons.tsx

```
@@ -0,0 +1,71 @@
+import { isMacOS } from "@tiptap/core";
+import { CornerUpLeft } from "lucide-react";
+import { Button } from "@/components/ui/button";
```

ESLint

[error] Replace `Button·}·from·"@/components/ui/button` with `KeyboardShortcut·}·from·"@/components/keyboardShortcut`

(prettier/prettier)